auth/index.js

```diff
@@ -0,0 +1,5 @@
+const {router} = require('./router');
+require('./strategies');
```

Comment:
This probably deserves a code comment, since we're requiring just to get the imperative code in strategies to run.